ultravox/training/evaluation.py

```diff
@@ -44,14 +45,36 @@ def dataset_infer(
     return ddp_utils.all_gather_list(eval_samples)
 
 
-def get_metric_name(ds_name: str, metric: str) -> str:
-    if ds_name == "boolq_in" and metric == "asr":
-        return "boolq__wer"
-    if ds_name == "boolq" and metric == "boolq":
-        return "boolq__correctness"
-    if metric == "instruct":
-        return f"{ds_name}__instruct_follow"
-    return f"{ds_name}__{metric}"
+@dataclasses.dataclass
+class EvalScenario:
+    name: str
+    dataset: str
+    metric: str
+    include_audio: bool = True
+    include_context: bool = True
+    new_tokens: Optional[int] = None
+
+
+EVAL_SCENARIOS = [
```

Comment:
Using the new dataclass allows us to move these scenarios into configs later on.
I was hesitant to do it yet though.

Reply:
Makes sense to me. Maybe alphabetize these scenarios?

Reply:
Alphabetized, but still kept the `text_only` evaluations separate.